wasm/Makefile.toml

```diff
@@ -0,0 +1,5 @@
+[tasks.wasm-build]
+description = "Build WASM target for deno_task_shell"
+command = "wasm-pack"
```

Comment:
Although my PR properly generates a WASM file, it's not clear to me what the best way to publish it would be given the new WASM support in Deno 2.1

I created an issue on JSR & `wasm-pack` to try and clarify this: https://github.com/rustwasm/wasm-pack/issues/1454